tests/unit/framework_utils/test_tf_feature_columns.py

```diff
@@ -21,4 +23,7 @@ def test_feature_column_utils():
     ]
 
     workflow, _ = nvtf.make_feature_column_workflow(cols, "target")
-    assert workflow.output_node.selector.names == ["vocab_1", "vocab_2", "target"]
+
+    schema = Schema(["vocab_1", "vocab_2", "target"])
+    workflow.fit_schema(schema)
```

Comment:
There's no dataset to fit here so I used a schema, but this seems like a bad user experience to have to manually fit. Maybe we can move that inside `make_feature_column_workflow` and fit it to an inferred schema automatically?